app/assets/stylesheets/components/_sortable.scss

```diff
@@ -14,10 +22,23 @@
 .sortable-dragging-svg {
   fill: #d4351c;
   margin-left: 2rem;
+  align-self: center;
 }
 
 .sortable-svg {
   fill: govuk-colour("mid-grey");
   margin-left: 2rem;
   align-self: center;
 }
+
+.nav-links {
+  display: flex;
+  align-items: center;
+  justify-content: left;
+  padding-top: 10px;
+}
+.divider {
+  display: inline-block;
+  margin: 0 0.5rem;
+  color: govuk-colour("mid-grey");
+}
```

Comment:
These are quite generic names so I wonder if it would make sense to nest them within `.sortable-list` or something?

Actually for the divider it's probably better to use a `border` on the nav-list elements.

Reply:
something like

```css
.nav-links {
  a + a { /* only links that are preceded by another link i.e. not the first */
    border-left: 1px solid govuk-colour("mid-grey");
    padding-left: 2rem;
  }
}
```

Reply:
Will give this a go 👍